src/module_PI/Raspberry_PI/main/PacketMessageTranslator.java

```diff
@@ -0,0 +1,110 @@
+package module_PI.Raspberry_PI.main;
+
+import java.util.*;
+
+public class PacketMessageTranslator {
+	
+	private Map<Integer, String> messageTable = new Hashtable<>();
+	private final int MAX_PACKET_DETAILS_INT = 55999;
+	private final int NUM_OF_BYTES_IN_PACKET_DETAILS = 3;
+	private final int NUM_OF_BYTES_IN_PACKET = 5;
+	private final int MAX_DETAILS_ONLY_INT = 999;
+	private final int MIN_FULL_PACKET_INT = 11001;
+	
+	public PacketMessageTranslator(){
+		buildTable();
+	}
+	
+	private void buildTable(){
+		messageTable.put(1, "Execute Command: Arm");
```

Comment:
This seems to be begging to be put into an ENUM. 

The advantage here would be then you could just iterate over the enum in a simple for loop (or Java 8 Stream). 

The advantage in testing could then be to access the enum to get the strings instead of hard coding (if you chose to do that - some school of thought say not to do that and to hard-code them in tests in case anyone changes the string accidentally).